_includes/v22.2/misc/s3-compatible-warning.md

```diff
@@ -0,0 +1,3 @@
+{{site.data.alerts.callout_danger}}
+Unlike Amazon S3, Google Cloud Storage, and Azure storage options, the usage of S3-compatible services is not actively tested by Cockroach Labs.
```

Comment:
Worth calling out examples? The ones we hear bout most are:

MinIO https://min.io
Red Hat Ceph https://docs.ceph.com/en/pacific/radosgw/s3/